src/OrbitGl/App.cpp

```diff
@@ -338,6 +339,18 @@ void OrbitApp::OnCaptureStarted(const orbit_grpc_protos::CaptureStarted& capture
         main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kInfo,
                                          absl::ZeroDuration(), "Capture started.");
 
+        orbit_version::Version capture_version{capture_started.orbit_version_major(),
+                                               capture_started.orbit_version_minor()};
+        orbit_version::Version current_version = orbit_version::GetVersionNumber();
+        if (capture_version > current_version) {
+          main_window_->AppendToCaptureLog(
+              MainWindowInterface::CaptureLogSeverity::kWarning, absl::ZeroDuration(),
+              absl::StrFormat(
+                  "Capture appears to be coming from the future: it was captured by Orbit version "
+                  "%d.%d, please open the capture using Orbit v%d.%d or above",
```

Comment:
This string gives me a "cartoonish" feeling ("coming from the future"), suggestion for a different one:
"The capture was taken with Orbit version %d.%d, which is higher than the current version. Please open the capture using Orbit v%d.%d or above."
What do you think?
Also add a final period as all the other messages in the capture log have it.

Reply:
Yeah, I feel the same.

Reply:
done.